src/db_options.rs

```diff
@@ -3074,6 +3074,30 @@ impl Options {
             ffi::rocksdb_options_set_allow_ingest_behind(self.inner, c_uchar::from(val));
         }
     }
+
+    /// The memory usage of memtable will report to this object. The same object
+    /// can be passed into multiple DBs and it will track the sum of size of all
+    /// the DBs. If the total size of all live memtables of all the DBs exceeds
+    /// a limit, a flush will be triggered in the next DB to which the next write
+    /// is issued, as long as there is one or more column family not already
+    /// flushing.
+    ///
+    /// If the object is only passed to one DB, the behavior is the same as
+    /// db_write_buffer_size. When write_buffer_manager is set, the value set will
+    /// override db_write_buffer_size.
+    ///
+    /// This feature is disabled by default. Specify a non-zero value
+    /// to enable it.
+    ///
+    /// Default: null
+    pub fn set_write_buffer_manager(&mut self, write_buffer_manager: &WriteBufferManager) {
+        unsafe {
+            ffi::rocksdb_options_set_write_buffer_manager(
+                self.inner,
+                write_buffer_manager.0.inner.as_ptr(),
+            );
+        }
```

Comment:
The `unsafe` block could use a comment. Is calling `rocksdb_options_set_write_buffer_manager` always sound?